pkg/actions/project.go

```diff
@@ -135,3 +139,106 @@ func ProjectRemoveConnection(c *cli.Context) {
 	fmt.Println(string(response))
 	os.Exit(0)
 }
+
+// ProjectList : Print the list of projects to the terminal
+func ProjectList(c *cli.Context) {
+	conID := strings.TrimSpace(strings.ToLower(c.String("conid")))
+
+	conInfo, conInfoErr := connections.GetConnectionByID(conID)
+	if conInfoErr != nil {
+		HandleConnectionError(conInfoErr)
+		os.Exit(1)
+	}
+
+	conURL, conErr := config.PFEOriginFromConnection(conInfo)
+	if conErr != nil {
+		HandleConfigError(conErr)
+		os.Exit(1)
+	}
+
+	projects, getAllErr := project.GetAll(http.DefaultClient, conInfo, conURL)
+	if getAllErr != nil {
+		HandleProjectError(getAllErr)
+		os.Exit(1)
+	}
+	if printAsJSON {
```

Comment:
could have a blank line before if printAsJSON, for consistency?

Reply:
couple new lines added